packages/@aws-cdk/aws-apigateway/lib/index.ts

```diff
@@ -1,2 +1,15 @@
+export * from './restapi';
+export * from './restapi-ref';
+export * from './resource';
+export * from './method';
+export * from './integration';
+export * from './deployment';
+export * from './stage';
+
+export * from './integrations/lambda';
```

Comment:
I would `export * from './integrations';` and have an `index.ts` in `./integrations/` that exports the various integrations. No hard feelings though.

Reply:
Done